src/pages/Landing/Landing.js

```diff
@@ -131,14 +164,56 @@ const Landing = ({ height, title, bg }) => {
             councilLabel: '',
           },
         });
-        // eslint-disable-next-line no-alert
-        alert(
-          // eslint-disable-next-line max-len
-          devEnvironment ? 'The region you have selected is not currently supported. We currently support Northeast, Midwest, and Southern Cover Crop Councils. Please try again!' : 'The region you have selected is not currently supported. We currently support Northeast Cover Crop Council. Please try again!',
-        );
       }
     }
-  }, [selectedRegion]);
+  }, [selectedState]);
+
+  const stateSelect = () => (
+    <List component="div" disablePadding>
+      <ListItem component="div">
+        <FormControl
+          variant="filled"
+          style={{ width: '100%' }}
+          sx={{ minWidth: 120 }}
+        >
+          <InputLabel>State</InputLabel>
+          <Select
+            variant="filled"
+            labelId="plant-hardiness-zone-dropdown-select"
+            id="plant-hardiness-zone-dropdown-select"
+            style={{
+              width: '100%',
+              textAlign: 'left',
+            }}
+            onChange={(e) => handleStateChange(e)}
+            value={selectedState.shorthand || ''}
+          >
+
+            {allStates.length > 0 && allStates.map((st, i) => (
+              <MenuItem value={st.shorthand} key={`Region${st}${i}`}>
+                {st.label?.toUpperCase()}
+              </MenuItem>
+            ))}
+          </Select>
+        </FormControl>
+      </ListItem>
+    </List>
+  );
+
+  useEffect(() => {
```

Comment:
there are two useEffects with the same dependencies

Reply:
pretty sure this whole useEffect can be deleted

Reply:
deleted in commit 01f8066